public/index.php

```diff
@@ -1,7 +1,41 @@
 <?php
+
+namespace GanttDashboard;
+
 use Phalcon\Di\FactoryDefault;
 
-error_reporting(E_ALL);
+/**
+ * APPLICATION ENVIRONMENT
+ */
+define('ENVIRONMENT', isset($_SERVER['APP_ENV']) ? $_SERVER['APP_ENV'] : 'development');
+
+/**
+ * ERROR REPORTING
+ */
+$environments = ['testing', 'production', 'development'];
+
+if (false === in_array(ENVIRONMENT, $environments)) {
+    header('HTTP/1.1 503 Service Unavailable.', true, 503);
+    echo 'The application environment is not set correctly.';
+    exit(1);
+}
+
+array_pop($environments);
```

Comment:
simplify this:
if (env is not development)
no errors

instead of removing the development from array. this might lead into confusion some developers